### PR TITLE
Footer & Queryoptions update

### PR DESCRIPTION
**Pull request by robertkowalski**
three commits:

```
footer: text update
```

```
remove update seq

update_seq does not work with couchdb 2. the footer text also was
not visible because of a css issue for months.
```

```
remove broken showStale

this was broken in multiple ways (option not visible and if option
visible changing it made no effect)

remove the feature as it has 0 users.
```

### CODE REVIEW

File: app/addons/documents/pagination/pagination.react.jsx [hunk 1]
```diff
@@ -139,16 +138,10 @@ define([
 
       pageNumber: function () {
         if (this.state.totalRows === 0) {
-          return <p> Showing 0 documents. </p>;
+          return <p>Showing 0 documents.</p>;
```
Comment (benkeen) [inline app/addons/documents/pagination/pagination.react.jsx]: Maybe drop the period?

File: app/addons/documents/queryoptions/actiontypes.js [hunk 2]
```diff
@@ -17,7 +17,6 @@ define([], function () {
     QUERY_TOGGLE_BETWEEN_KEYS: 'QUERY_TOGGLE_BETWEEN_KEYS',
     QUERY_UPDATE_BETWEEN_KEYS: 'QUERY_UPDATE_BETWEEN_KEYS',
     QUERY_UPDATE_BY_KEYS: 'QUERY_UPDATE_BY_KEYS',
-    QUERY_TOGGLE_UPDATE_SEQ: 'QUERY_TOGGLE_UPDATE_SEQ',
     QUERY_TOGGLE_DESCENDING: 'QUERY_TOGGLE_DESCENDING',
     QUERY_TOGGLE_REDUCE: 'QUERY_TOGGLE_REDUCE',
     QUERY_UPDATE_SKIP: 'QUERY_UPDATE_SKIP',
```
Comment (benkeen) [inline app/addons/documents/queryoptions/actiontypes.js]: I think we can remove QUERY_SHOW_STALE from this file too.

**Comment by benkeen** [comment]
Other than the minor things above, code looks good + works fine. I'm all for removing features that are simply not used. I wonder if people will notice that the stale option is no longer there and pipe up. 

But I think +1.

**Comment by robertkowalski** [comment]
fixed and merged :)

**Pull request closed without merging**